extensions/blocks/button/constants.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable wpcalypso/import-docblock */
+/**
+ * WordPress dependencies
+ */
+import { __experimentalUseGradient as useGradient } from '@wordpress/block-editor';
```

Comment:
Nitpick: Should this be capitalized? `UseGradient`

Reply:
It shouldn't: `useGradient` is a hook, like `useEffect`.